backend/open_api_spec.yml

```diff
@@ -0,0 +1,683 @@
+openapi: 3.0.0
+info:
+servers:
+paths:
+  /curation/auth/key:
+    post:
+      summary: Generate the curator's API key -- for use with Data Portal UI
+      security:
+        - cxguserCookie: []
+      responses:
+        "201":
+          description: Created
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  key:
+                    type: string
+                    required: true
+                  id:
+                    type: string
+                    required: true
+        "401":
+          $ref: "#/components/responses/401"
+        "400":
+          $ref: "#/components/responses/400"
+    get:
+      summary: Fetch the identifier of the curator's active API key -- for use with Data Portal UI
+      security:
+        - cxguserCookie: []
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: string
+        "401":
+          $ref: "#/components/responses/401"
+        "404":
+          $ref: "#/components/responses/404"
+    delete:
+      summary: Delete the curator's API key -- for use with Data Portal UI
+      security:
+        - cxguserCookie: []
+      responses:
+        "202":
+          $ref: "#/components/responses/202"
+        "401":
+          $ref: "#/components/responses/401"
+
+  /curation/auth/token:
+    post:
+      summary: Generate an access token
+      description: >-
+        Returns a bearer access token that must be passed as a parameter to requests that
+        require authorization such as creating a new Collection.
+      security:
+        - apiKey: []
+      responses:
+        "201":
+          description: Created
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  access_token:
+                    type: string
+        "401":
+          $ref: "#/components/responses/401"
+
+  /curation/collections:
```

Comment:
Consider whether this endpoint is needed. This is effectively a dump of all of our database, and will scale poorly in the future (even now, that's a few megabytes of data). For comparison, the FE only receives a subset of this data on load.

An alternative is to add pagination.

Reply:
Yep, it'll be a good thing to keep tabs on as we move forward.

For now, it is desired for our curators. We'll see about adding pagination if it's absolutely necessary or if we have a little spare time at the end.